pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu

```diff
@@ -129,7 +129,7 @@ ENV PATH="$OPEN_MPI_PATH/bin:$PATH"
 ENV LD_LIBRARY_PATH=$OPEN_MPI_PATH/lib/:$EFA_PATH/lib/:$LD_LIBRARY_PATH
 
 RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value \
- && curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && curl -L -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-py37_4.10.3-Linux-x86_64.sh \
```

Comment:
Can you add comments here describing the version pin and also why there is "py37" in this specific sh script?

Reply:
Could you also add a few words into the PR description. Why are you switching to anaconda repo and why do you pin the script version

Reply:
added in dockerfile, will add it in description also